app/src/main/sentences/fr/search.dslf

```diff
@@ -0,0 +1,2 @@
+search: low
+cherche|cherchez|chercher|recherche|recherchez|rechercher|consulte|consultez|consulter ((ce que)|(ce qu'est)|si)? .what. est? (sur internet|l'internet|le web)?;
```

Comment:
I also corrected a small mistake: the whole `(le web)` should be an alternative to `internet`, not just `le` ;-)
```suggestion
cherche|cherchez|chercher|recherche|recherchez|rechercher|consulte|consultez|consulter ((ce que)|(ce qu est)|si)? .what. est? (sur internet|(l internet)|(le web))?;
```